src/zc/buildout/easy_install.py

```diff
@@ -1946,3 +1947,17 @@ def sort_working_set(ws, eggs_dir, develop_eggs_dir):
     sorted_paths.extend(egg_paths)
     sorted_paths.extend(other_paths)
     return pkg_resources.WorkingSet(sorted_paths)
+
+
+NOT_PICKED_AND_NOT_ALLOWED = """\
+Picked: {name} = {version}
+
+The `{name}` egg does not have a version pin and allow-picked-versions = false.
+
+To resolve this, add
+
+    {name} = {version}
+
+to the [versions] section,
+
+OR set allow-picked-versions = true."""
```

Comment:
I would put the the sentence after `OR` not on a single line, but just after the comma ... but that is absolutely a matter of personal style :-)